fractal-puppy/Engine
# Enemy scripts fix

- Fixed enemies hitboxes.
- Updated Graveyard scene to include more enemies.
- Added two scenes: EnemiesSample (includes one enemy of each type ready to add them to any scene) and EnemiesTest (to test only the enemies with the player).

To test:
- Load "EnemiesTest" scene and check that the damage done to the player is only inflicted when they are attacking. Check that everything related to combat works: life bars, player life, enemies highlights, etc...
- Load "Level0-TheGraveyard" scene and check that the new enemies included work and nothing is broken.
